dubbo-common/src/main/java/org/apache/dubbo/config/AbstractInterfaceConfig.java

```diff
@@ -252,7 +252,7 @@ protected void postProcessAfterScopeModelChanged(ScopeModel oldScopeModel, Scope
         }
         if (CollectionUtils.isNotEmpty(this.registries)) {
             this.registries.forEach(registryConfig -> {
-                if (registryConfig.getScopeModel() != applicationModel) {
+                if (registryConfig != null && registryConfig.getScopeModel() != applicationModel) {
```

Comment:
**Security Issue**: Null check should be used to prevent potential NullPointerException when accessing registryConfig.getScopeModel(). <br> **Fix**: Add null check for registryConfig before accessing its methods to ensure null safety. <br> **Code Suggestion**: 
 ```
 -                if (registryConfig.getScopeModel() != applicationModel) {
 +                if (registryConfig != null && registryConfig.getScopeModel() != applicationModel) {
 ```